src/app/dashboard/_components/smart-notes/CommandMenu.tsx

```
@@ -177,102 +183,249 @@ export function CommandMenu({ onSelect, onClose, searchTerm = '', position }: Co
+          preview: `Fifth AI-generated idea for ${postTypeData.description}`,
+          type: 'block',
+          template: generateTemplate(selectedPlatform, selectedPostType, postTypeData.description),
+          metadata: { type: 'idea', value: true },
+        }
+      ];
+      
+      // Filter AI prompts by search term if provided
+      const filteredAiPrompts = searchTerm
+        ? aiPromptCommands.filter(cmd =>
+            cmd.label.toLowerCase().includes(searchTerm.toLowerCase()) ||
+            (cmd.preview && cmd.preview.toLowerCase().includes(searchTerm.toLowerCase()))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)